### PR TITLE
[FEA] `Kurtosis` and `skew` for dataframe

### PR DESCRIPTION
**Pull request by mzegar**
From my testing, this should always return `never` if provided an invalid DataFrame containing non numeric Series.

### CODE REVIEW

File: modules/cudf/test/cudf-data-frame-tests.ts [hunk 1]
```diff
@@ -832,3 +856,8 @@ test(`DataFrame.dropDuplicates("first", true, true, ['a'])`, () => {
   expect([...result.get('a')]).toEqual([null, 1, 3, 4]);
   expect([...result.get('b')]).toEqual([null, 5, 8, 2]);
 });
+
+// Typescript does not allow us to throw a compile-time error if
+// the result of a method is `never`. Instead, let's just verify
+// the result is `never`.
+function _verifySeriesIsNever(_: never) { return _; }
```
Comment (AjayThorve) [inline modules/cudf/test/cudf-data-frame-tests.ts:863]: shouldn't this be `_verifySeriesIsNever(_: Series<never>)` ?

Comment (mzegar) [reply modules/cudf/test/cudf-data-frame-tests.ts:863]: I need to double check the difference, but technically what I return is just `never`, whereas this https://github.com/rapidsai/node-rapids/pull/240 has `Dataframe<never>`. Should check out the difference.

Comment (AjayThorve) [reply modules/cudf/test/cudf-data-frame-tests.ts:863]: I was referring to the result of `skew` [here](https://github.com/mzegar/node-rapids/blob/df-skew-kurtosis/modules/cudf/src/data_frame.ts#L1628), where you were returing `Series<never>`.

But yes, if #240 works, then we can ignore this

Comment (mzegar) [reply modules/cudf/test/cudf-data-frame-tests.ts:863]: ah you're right, I'll double check this tomorrow. For whatever reason `ts` was inferring it as `never`, I'll look into this.

**Review by AjayThorve — APPROVED** [review]
LGTM

File: modules/cudf/test/cudf-data-frame-tests.ts (hunk 1 above)
Comment (mzegar) [reply modules/cudf/test/cudf-data-frame-tests.ts:865]: Verified that this returns `never`.

**Pull request merged**